sdk/resourcemanager/Proto.Client/compute/Extensions/ApiVersionsExtensions.cs

```diff
@@ -1,20 +0,0 @@
-using Azure.ResourceManager.Core;
```

Comment:
where did this class go?

Reply:
Do we need this? I just need the already made xxxRestApiVersions which calls the GetOverrideObject method in AzureReousceManagerClientOptions